tests/keras/test_keras_model_export.py

```diff
@@ -210,21 +210,30 @@ def _import_module(name, **kwargs):
             assert x == b
 
 
-@pytest.mark.skip("Unskip this test once https://github.com/h5py/h5py/issues/1732 is fixed")
-@pytest.mark.parametrize("build_model", [model, tf_keras_model])
+@pytest.mark.parametrize(
+    "build_model,save_format",
+    # Unskip these parameters once https://github.com/h5py/h5py/issues/1732 is fixed
```

Comment:
```suggestion
    # TODO: Unskip these parameters once https://github.com/h5py/h5py/issues/1732 is fixed
```

This is a nit but can you add `TODO` here to make it easier to notice this later?